app/src/main/java/com/spendster/presentation/moreAboutYou/MoreAboutYouActivity.java

```diff
@@ -105,6 +105,8 @@ private void takePhotoFromCamera() {
                 String[] permissionRequested = {Manifest.permission.CAMERA};
                 requestPermissions(permissionRequested, CAMERA_REQUEST_CODE);
             }
+        }else {
+            Toast.makeText(this, "Error", Toast.LENGTH_SHORT).show();
```

Comment:
@SimonMaha It would be nice to show the text describing the error, not just "error" placeholder